### ISSUE TITLE
Loki Output Failing

### ISSUE THREAD

**Issue by egallis31** [unpoller/unpoller#696]
Potential issue with the timestamp conversion, which causes Loki Output to fail, no configuration changes and no logs past, started on v2.11.0. Each auto upgrade to latest stable for both unpoller and Unifi Network, last update to Unifi Network was 3/19 + unpoller following releases during updates. 

Thought that it was intially some type of authentication issue as I recently changed the local user role on the Unifi system, but reverted the permissions and still receiving the error

```
[ERROR] event fetch for Loki failed: unifi.GetIDS(): json: cannot unmarshal string into Go struct field IDS.data.timestamp of type int64: error initializing inputs
```

unpoller Version: v2.11.1
Unifi OS Version: v3.2.12
Unifi Network Version: 8.1.113

Config:

```
  UP_INFLUXDB_DISABLE: 'true'
  UP_LOKI_DISABLE: 'false'
  UP_LOKI_URL: http://loki-gateway.loki:80
  UP_POLLER_DEBUG: 'true'
  UP_PROMETHEUS_DISABLE: 'false'
  UP_PROMETHEUS_NAMESPACE: unifipoller
  UP_UNIFI_DEFAULT_SAVE_ALARMS: 'true'
  UP_UNIFI_DEFAULT_SAVE_ANOMALIES: 'true'
  UP_UNIFI_DEFAULT_SAVE_DPI: 'true'
  UP_UNIFI_DEFAULT_SAVE_EVENTS: 'true'
  UP_UNIFI_DEFAULT_SAVE_IDS: 'true'
  UP_UNIFI_DEFAULT_SAVE_SITES: 'true'
  UP_UNIFI_DYNAMIC: 'false'
```

Log Output:

```
2024/04/07 16:37:23.382310 updateweb.go:222: [DEBUG] Collecting controller events: https://unifi:443 ()
2024/04/07 16:37:23.382408 updateweb.go:222: [DEBUG] Requesting https://10.2.0.254:443/proxy/network/api/stat/sites, with params: false, cookies: 1
2024/04/07 16:37:23.401049 updateweb.go:222: [DEBUG] Requested https://10.2.0.254:443/proxy/network/api/stat/sites: elapsed 19ms, returned 2361 bytes
2024/04/07 16:37:23.401195 updateweb.go:222: [DEBUG] Found 1 site(s): default
2024/04/07 16:37:23.401203 updateweb.go:222: [DEBUG] Collecting controller IDS data: https://unifi:443 ()
2024/04/07 16:37:23.401207 updateweb.go:222: [DEBUG] Polling Controller for IDS Events, site Default (default)
2024/04/07 16:37:23.401263 updateweb.go:222: [DEBUG] Requesting https://10.2.0.254:443/proxy/network/api/s/default/stat/ips/event, with params: true, cookies: 1
2024/04/07 16:37:23.867163 updateweb.go:222: [DEBUG] Requested https://10.2.0.254:443/proxy/network/api/s/default/stat/ips/event: elapsed 466ms, returned 2568507 bytes
2024/04/07 16:37:23.913645 logger.go:32: [ERROR] event fetch for Loki failed: unifi.GetIDS(): json: cannot unmarshal string into Go struct field IDS.data.timestamp of type int64: error initializing inputs

```

**Comment by davidnewhall** [comment]
Thanks for your bug report. Ubiquiti recently broke API compatibility. Still trying to figure out which fields, and we'll try to get it fixed asap.

**Comment by hanley-development** [comment]
Just updated the UDM Pro today and started getting these. I regret upgrading, but really wanted to start using the DNS functionality.

```
> 2024/06/10 00:08:16.673992 updateweb.go:222: [DEBUG] Collecting controller events: https://192.168.0.1 ()
> 2024/06/10 00:08:16.674039 updateweb.go:222: [DEBUG] Requesting https://192.168.0.1/proxy/network/api/stat/sites, with params: false, cookies: 1
> 2024/06/10 00:08:16.706913 updateweb.go:222: [DEBUG] Requested https://192.168.0.1/proxy/network/api/stat/sites: elapsed 33ms, returned 2189 bytes
> 2024/06/10 00:08:16.707009 updateweb.go:222: [DEBUG] Found 1 site(s): default
> 2024/06/10 00:08:16.707015 updateweb.go:222: [DEBUG] Collecting controller IDS data: https://192.168.0.1 ()
> 2024/06/10 00:08:16.707018 updateweb.go:222: [DEBUG] Polling Controller for IDS Events, site Default (default)
> 2024/06/10 00:08:16.707036 updateweb.go:222: [DEBUG] Requesting https://192.168.0.1/proxy/network/api/s/default/stat/ips/event, with params: true, cookies: 1
> 2024/06/10 00:08:18.038562 updateweb.go:222: [DEBUG] Requested https://192.168.0.1/proxy/network/api/s/default/stat/ips/event: elapsed 1.332s, returned 8755239 bytes
> 2024/06/10 00:08:18.110532 logger.go:32: [ERROR] event fetch for Loki failed: unifi.GetIDS(): json: cannot unmarshal string into Go struct field IDS.data.timestamp of type int64: error initializing inputs
```

**Comment by platinummonkey** [comment]
this should be fixed in v2.12.0
